content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,156 @@
+
+    {{< text bash >}}
+    $ CLUSTER_NAME="<the name of your cluster>"
+    {{< /text >}}
+
+1. Create a working directory for files generated in this documentation.
+ 
+    {{< text bash >}}
+    $ mkdir -p "${HOME}"/"${CLUSTER_NAME}"
+    {{< /text >}}
+
+1. Execute the following commands on each cluster in the mesh where a virtual
+    machine connection should be established. See
+    [Certificate Authority (CA) certificates](/docs/tasks/security/cert-management/plugin-ca-cert/)
+    for more details on configuring an external CA.
+
+    {{< text bash >}}
+    $ make -C sample/certs NAME="${CLUSTER_NAME}" NAMESPACE="vm" "${CLUSTER_NAME}"-certs-wl
+    $ cp -a samples/certs/"${CLUSTER_NAME}"/ca-cert.pem "${HOME}"/"${CLUSTER_NAME}"
+    $ cp -a samples/certs/"${CLUSTER_NAME}"/ca-key.pem "${HOME}"/"${CLUSTER_NAME}"
+    $ cp -a samples/certs/"${CLUSTER_NAME}"/root-cert.pem "${HOME}"/"${CLUSTER_NAME}"
+    $ cp -a samples/certs/"${CLUSTER_NAME}"/workload-cert-chain.pem "${HOME}/"${CLUSTER_NAME}"/cert-chain.pem
+    $ kubectl create namespace istio-system
+    $ kubectl create secret generic cacerts -n istio-system \
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/ca-cert.pem \
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/ca-key.pem \
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/root-cert.pem \
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/cert-chain.pem
+    {{< /text >}}
+
+1. Generate a `cluster.env` configuration file that informs the virtual machine
+   deployment which network CIDR to capture and redirect to the Kubernetes
+   cluster:
+
+    {{< text bash >}}
+    $ ISTIO_SERVICE_CIDR=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
+    $ echo -n ISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR > "${HOME}"/"${CLUSTER_NAME}"/cluster.env
+    {{< /text >}}
+
+1. Configure a select set of ports for exposure from the virtual machine. If you do not
+   apply this optional step, all outbound traffic on all ports is sent to the
+   Kubernetes cluster. You may wish to send some traffic on specific ports to
+   other destinations. This example shows enabling ports `3306` and `8080`
+   for capture by Istio virtual machine integration and transmission to Kubernetes.
+   All other ports are sent over the default gateway of the virtual machine.
+
+    {{< text bash >}}
+    $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> "${HOME}"/"${CLUSTER_NAME}"/cluster.env
+    {{< /text >}}
+
+1. Is this step needed if we always assume there is a load balancer? Try removing.
+    The ingress gateway port for Istiod is automatically programmed to 15012 throughout
+    the Istio software. Record the ISTIO_PILOT_PORT address in cluster.env.
+
+    {{< text bash >}}
+    $ echo "ISTIO_PILOT_PORT=15012" >> ${HOME}/${CLUSTER_NAME}"/cluster.env
+    {{< /text >}}
+
+
+1. Install Istio with virtual machine integration features enabled:
```

Comment:
I suggest linking to the MC install instructions - mesh expansion should be using the same setup.

Reply:
That seems reasonable to me as well, if that helps. As @sdake and I discussed offline previously, these will (soon) become part of the same installation guide anyway.

Reply:
the MC instructions are not yet on the website. @rcaballeromx is using the docs workflow for those in this document: https://docs.google.com/document/d/1wzoFqLHw-dEoKPVi1w5tlckWTn78qpYNbC5fEwp-Mw4/edit. The docs team has agreed for now, to keep these two concepts independent, even though they are related in many ways.

Cheers
-steve